src/main/java/org/jitsi/jibri/api/http/HttpApi.kt

```diff
@@ -0,0 +1,93 @@
+package org.jitsi.jibri.api.http
```

Comment:
Someone once yelled at me for not having licenses in my files.

Reply:
yeah have to do all those